supervisor/resolution/evaluations/container.py

```diff
@@ -57,23 +73,28 @@ async def evaluate(self) -> None:
                 self.sys_resolution.evaluate.cached_images.add(tag)
 
                 # Evalue system
-                image_name = tag.partition(":")[0].split("/")[-1]
-                if (
-                    any(
-                        image_name.startswith(deny_name)
-                        for deny_name in DOCKER_IMAGE_DENYLIST
-                    )
-                    and image_name not in self._images
-                ):
+                image_name = tag.partition(":")[0]
+                if image_name not in self.known_images:
                     self._images.add(image_name)
+                    if any(
+                        image_name.split("/")[-1].startswith(deny_name)
+                        for deny_name in DOCKER_IMAGE_DENYLIST
+                    ):
+                        _LOGGER.error(
+                            "Found image in deny-list '%s' on the host", image_name
+                        )
+                        self.sys_resolution.unhealthy = UnhealthyReason.DOCKER
```

Comment:
We should not drop `UnsupportedReason.CONTAINER`. These containers are known to affect a system bad, UnsupportedReason.DOCKER is more in case of a docker directly issue and not a 3party trouble.

Reply:
We are not dropping it.